services/liquibase/main-db/changelog/changelogs/2025-04-19-1840-init-schema.yaml

```diff
@@ -0,0 +1,170 @@
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: ingredient_id, type: uuid }
+              - column: { name: tag_id, type: uuid }
+
+        # recipe_ingredients
+        - createTable:
+            tableName: recipe_ingredients
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: recipe_id, type: uuid }
+              - column: { name: ingredient_id, type: uuid }
+
+        # users
+        - createTable:
+            tableName: users
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: first_name, type: varchar(200) }
+              - column: { name: second_name, type: varchar(200) }
+              - column: { name: last_name, type: varchar(200) }
+              - column: { name: email_address, type: varchar(200) }
+              - column: { name: password_hash, type: varchar(512) }
+
+        # user_fridge_ingredients
+        - createTable:
+            tableName: user_fridge_ingredients
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: user_id, type: uuid }
+              - column: { name: ingredient_id, type: uuid }
+
+        # administrators
+        - createTable:
+            tableName: administrators
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false } }
+
+        # review
+        - createTable:
+            tableName: review
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: recipe_id, type: uuid }
+              - column: { name: user_id, type: uuid }
+              - column: { name: rating, type: float(2) }
+              - column: { name: description, type: varchar(2000) }
+              - column: { name: has_photos, type: boolean }
+
+        # photo_urls
+        - createTable:
+            tableName: photo_urls
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false } }
+              - column: { name: photo_url, type: varchar(512) }
+
+        # user_dietary_restrictions
+        - createTable:
+            tableName: user_dietary_restrictions
+            columns:
+              - column: { name: id, type: uuid, constraints: { primaryKey: true, nullable: false }, defaultValueComputed: gen_random_uuid() }
+              - column: { name: ingredient_tags_id, type: uuid }
```

Comment:
ingredient_tags_id -> ingredient_category_id, also should be a foreign key pointing to ingredient_categories table